### DISCUSSION TITLE
Hosting on AWS Lambda

### DISCUSSION THREAD

**Discussion by ctf05** [BrokenSource/DepthFlow#34] [Q&A]
First of all, great tool. It's amazing. But I've spent the last 5 days trying to cram it on AWS Lambda. I have many problems with glfw and writing to read only directories. Is there any easier way to handle this?

**Comment by Tremeschin** [comment]
I'm not sure if it's possible to use OpenGL acceleration on lambda as I never used the service myself, but this docker guide I wrote should give some insights on what needs to happen for it: https://brokensrc.dev/get/docker/

as for glfw, simply use `WINDOW_BACKEND=headless depthflow (...)` or initialize a scene as `DetphScene(backend="headless")`, see also the [`custom.py`](https://github.com/BrokenSource/DepthFlow/blob/main/Examples/Custom.py) file for managing it

ideally, set the output video to an absolute path, but relative works also, and the scene returns a list of rendered videos, as seen in the current [webui](https://github.com/BrokenSource/DepthFlow/blob/5d6108e196c6d2c8d4872f13e057b598bd52d008/DepthFlow/Webui.py#L75) implementation
